parse_weather_data.py

```diff
@@ -0,0 +1,48 @@
+from weather_reading import WeatherReading
+
+
+class WeatherParser:
+    def parse_weather_file(self, file_path):
+        weather_readings = []
+        try:
```

Comment:
This try except block added too much indentation which make it difficult for one to read.